website/docs/getting-started/settings.md

```diff
@@ -0,0 +1,25 @@
+---
+id: settings
+title: Settings
+sidebar_position: 4
+---
+
+# LanSuite generic config settings
+
+Below you find an explanation of the various config settings you find in the LanSuite Admin Area
```

Comment:
I'm with you on the duplication of "find", but to my knowledge this is otherwise (more) correct as-is.

"will" is not correct, unless that is meant in regards to the documentation not being complete _yet_. 
But there is documentation to be found when the user reads this, thus technically present tense is correct.

"encounter" is normally used in scope of an unexpected situation that occurrs. 
But fine to replace that with e.g. "see", to prevent duplicate use of find.
Or just remove it alltogether: "config settings in the LanSuite Admin Area"

Interpunctuation is not really needed, as there is no following sentence to separate from (but no issue to add that)

Reply:
I use grammarly.com for a lot of English communication, and mostly, it has a better "readable" result.
However, I am fine as it is.